docs/docsite/rst/network/getting_started/network_roles.rst

```diff
@@ -0,0 +1,37 @@
+
+.. _using_network_roles:
+
+******************************************
+Use Ansible network roles
+******************************************
+
+The Ansible Network team develops and supports a set of `network-related roles <https://galaxy.ansible.com/ansible-network>`_ on Galaxy. You can use these roles to jumpstart your network automation efforts. These roles are updated approximately every two weeks to give you access to the latest Ansible networking content.
+
+These roles come in the following categories:
+
+* **User roles** - You use these roles, such as `config_manager <https://galaxy.ansible.com/ansible-network/config_manager>`_ and `cloud_vpn <https://galaxy.ansible.com/ansible-network/cloud_vpn>`_, directly in your playbooks. These roles are platform/provider agnostic, allowing you to use the same roles and playbooks across different network platforms or cloud providers.
+* **Platform provider roles** - Network user roles depend on these provider roles to implement their functions. For example, the `config_manager <https://galaxy.ansible.com/ansible-network/config_manager>`_ role  uses the  `cisco_ios <https://galaxy.ansible.com/ansible-network/cisco_ios>`_ provider role to implement tasks on Cisco IOS network devices.
```

Comment:
I'd describe this in a more provider-role-centric way. Maybe something like "Provider roles translate between the user roles and the various network OSs, each of which has a different API. Each provider role accepts input from any user role and translates it for a specific network OS. For example, the `cisco_ios` provider role implements tasks on Cisco IOS network devices." Because that provider role accepts input from any user role, right? not just from `config_manager`.